src/calculateRentalCost.js

```diff
@@ -28,7 +28,20 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const basicPrice = 40;
+  const firstDiscount = 20;
+  const secondDiscount = 50;
```

Comment:
Constants must be in upper case and snake case.

Example: `BASIC_PRICE` etc.

Reply:
Hi, I did it)